pkg/registry/core/pod/rest/subresources.go

```diff
@@ -193,8 +191,8 @@ func (r *PortForwardREST) Connect(ctx context.Context, name string, opts runtime
 
 func newThrottledUpgradeAwareProxyHandler(location *url.URL, transport http.RoundTripper, wrapTransport, upgradeRequired, interceptRedirects bool, responder rest.Responder) *proxy.UpgradeAwareHandler {
```

Comment:
The `intercceptRedirects` parameter is no longer used, and can be removed.

Reply:
Updated. Thanks for reviewing.